src/frontend/src/lib/components/buy/BuyModal.svelte

```diff
@@ -11,7 +11,7 @@
 		>{isOnRamperDev ? $i18n.buy.text.buy_dev : $i18n.buy.text.buy}</svelte:fragment
 	>
 
-	<div class="stretch overflow-hidden">
+	<div class="stretch">
```

Comment:
`overflow-hidden` was introduced to fix the broken style, sorry for the bug.

that said, if we remove this, then the style is broken again.

Reply:
I am removing it only here in the BuyModal; I tested the other modals and it doesn't seem to be affecting those

Just to confirm, does that answer the question?

Reply:
That wasn't a question. 😉

See your video, by removing the style you indeed fix the scrolling but you break the style fix provided in #3220 (which unfortunately broke the scroll, as I said, sorry for that).

Reply:
Now I see what you mean, I took time (tks for the offline explanation) eheh!

After some tries, have a look at last commit


https://github.com/user-attachments/assets/fb5f52c9-b839-4215-b06c-3175512db31e

Reply:
COOL